users/templates/users/reset_password.html

```diff
@@ -0,0 +1,27 @@
+{% extends 'base.html' %}
+{% load static %}
+{% load crispy_forms_tags %}
+
+{% block content %}
+    <div class="container text-center mt-4 text-white">
+        <h2>Welcome to the Password Reset Page</h2>
+        <p>
+            Forgot your password? Please enter the email address you used to register
+            with us, and we will send you a link to reset your password.
+        </p>
+    </div>
+    <div class="d-flex justify-content-center mt-4">
+        <div class="p-4 m-4 text-white">
+            <form action="" method="POST" class="text-center">
+                {% csrf_token %}
+                {{ field.errors }}
```

Comment:
This won't work. You are using field.errors not "non_field_errors" as was requested. Please check registration and just copy that code for non_field_errors here. 
Also I don't even know if field.errors exist.